src/daemon.rs

```diff
@@ -392,11 +392,3 @@ pub fn take_snapshot(logger: &Logger, rpc_addr: SocketAddr) -> Result<()> {
     info!(logger, "The frugalos server has taken snapshot");
     Ok(())
 }
-
-fn rpc_write_timeout() -> Duration {
-    if let Some(x) = option_env!("FRUGALOS_RPC_WRITE_TIMEOUT_SEC").and_then(|s| s.parse().ok()) {
```

Comment:
`FRUGALOS_RPC_WRITE_TIMEOUT_SEC` is no longer supported. I beleve noone use this feature :-)

Reply:
👍 
 I think this removal will cause no problem.